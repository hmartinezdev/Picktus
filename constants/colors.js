export default {
    main: '#f7dab7',
    secondary: '#3b3937',
    lightGreen: '#b3e099',
    blue: '#9fcbe7',
    green: '#75ab88'
}