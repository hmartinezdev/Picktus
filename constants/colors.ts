export default {
  primary: '#2d3047',
  secondary: '#FAC9B8',
  lightGreen: '#b3e099',
  blue: '#9fcbe7',
  green: '#75ab88',
};
