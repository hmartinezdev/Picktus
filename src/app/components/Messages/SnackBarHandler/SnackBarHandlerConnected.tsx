import { IReduxState } from '@store/reducers';
import { dismissSnackBar } from '@store/reducers/message/message-actions';
import { connect } from 'react-redux';
import SnackBarHandler from './SnackBarHandler';
import { IReduxStateProps } from './SnackBarHandler.type';

const mapStateToProps = (state: IReduxState): IReduxStateProps => {
  return {
    snackbars: state.message.snackbars,
  };
};

export default connect(
  mapStateToProps,
  { dismissSnackBar }
)(SnackBarHandler);
