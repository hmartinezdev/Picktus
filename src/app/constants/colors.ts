export default {
  black: '#151614',
  blue: '#9fcbe7',
  error: '#982649',
  green: '#75ab88',
  lightGreen: '#b3e099',
  primary: '#2d3047',
  secondary: '#FAC9B8',
  warning: '#E37E5B',
  white: '#F8F8F7',
};
