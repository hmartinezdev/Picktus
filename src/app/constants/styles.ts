export const borderRadius = '3px';
export const fontFamily = "'Josefin Sans', sans-serif";
export const boxShadow = '11px 10px 34px -7px rgba(0, 0, 0, 0.49)';
