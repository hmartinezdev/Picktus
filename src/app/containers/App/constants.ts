export const config = {
  apiKey: 'AIzaSyAVQ-Abn_Sf0z18jPeW_mKhLOwwGfWn_40',
  authDomain: 'picktus-76101.firebaseapp.com',
  databaseURL: 'https://picktus-76101.firebaseio.com',
  projectId: 'picktus-76101',
  storageBucket: 'picktus-76101.appspot.com',
  messagingSenderId: '1077268436650',
};
