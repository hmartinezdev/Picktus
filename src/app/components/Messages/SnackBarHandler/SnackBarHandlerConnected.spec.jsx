import { mount, shallow } from 'enzyme';
import React from 'react';
import configureStore from 'redux-mock-store';
import SnackBarHandlerConnected from './';
import { config } from './constants';

let wrapper;
let store;

// create any initial state needed
const initialState = {
  message: {
    snackbars: [],
  },
};

// here it is possible to pass in any middleware if needed into //configureStore
const mockStore = configureStore();

const setup = (props = {}) => shallow(<SnackBarHandlerConnected {...props} />);

describe('<SnackBarHandlerConnected />', () => {
  beforeEach(() => {
    store = mockStore(initialState);
    wrapper = setup({ store });
  });

  test('should render properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
