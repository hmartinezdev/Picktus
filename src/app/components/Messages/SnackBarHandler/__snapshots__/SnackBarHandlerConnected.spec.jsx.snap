// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SnackBarHandlerConnected /> should render properly 1`] = `
<SnackBarHandler
  dismissSnackBar={[Function]}
  snackbars={Array []}
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
  storeSubscription={
    Subscription {
      "listeners": Object {
        "clear": [Function],
        "get": [Function],
        "notify": [Function],
        "subscribe": [Function],
      },
      "onStateChange": [Function],
      "parentSub": undefined,
      "store": Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "unsubscribe": [Function],
    }
  }
/>
`;
